packages/rum-core/src/boot/rumPublicApi.ts

```diff
@@ -223,6 +228,10 @@ export function makeRumPublicApi(
       addTimingStrategy(name, time as RelativeTime | TimeStamp | undefined)
     }),
 
+    addFeatureFlags: monitor((featureFlags: object) => {
+      addFeatureFlagsStrategy(featureFlags as Context)
+    }),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏don't add this method if the FF is disabled. Also, make sure it doesn't show up in public TS typings